src/features/side_by_side.rs

```diff
@@ -380,23 +381,27 @@ fn right_pad_left_panel_line(
     // Pad with (maybe painted) spaces to the panel width.
     let text_width = ansi::measure_text_width(&panel_line);
     let panel_width = config.side_by_side_data.left_panel.width;
-    if text_width < panel_width {
-        let fill_style = get_right_fill_style_for_left_panel(
-            panel_line_is_empty,
-            line_index,
-            &diff_style_sections,
-            state,
-            background_color_extends_to_terminal_width,
-            config,
-        );
-        panel_line.push_str(
-            &fill_style
-                .paint(" ".repeat(panel_width - text_width))
-                .to_string(),
-        );
-    } else if text_width > panel_width {
-        *panel_line =
-            ansi::truncate_str(panel_line, panel_width, &config.truncation_symbol).to_string();
+    match text_width.cmp(&panel_width) {
+        Ordering::Less => {
+            let fill_style = get_right_fill_style_for_left_panel(
+                panel_line_is_empty,
+                line_index,
+                &diff_style_sections,
+                state,
+                background_color_extends_to_terminal_width,
+                config,
+            );
+            panel_line.push_str(
+                &fill_style
+                    .paint(" ".repeat(panel_width - text_width))
+                    .to_string(),
+            );
+        }
+        Ordering::Greater => {
+            *panel_line =
+                ansi::truncate_str(panel_line, panel_width, &config.truncation_symbol).to_string();
+        }
+        std::cmp::Ordering::Equal => {}
```

Comment:
I'm not sure this is an improvement --I think the resulting code is harder to read. But I also didn't know it was an idiom/recommended style. Do you have an opinion?

Reply:
Yeah, I understand it may be harder to read. I wanted to do this in a separate PR, but laziness..

I can disable this warning locally for the function :)

Reply:
Yes, let's disable the linter on this one. Other than this one I'm on board with it all!